docs/en/observability/observability-ai-assistant.asciidoc

```diff
@@ -0,0 +1,101 @@
+[[obs-ai-assistant]]
+= Observability AI Assistant
+
+preview::[]
+
+The Observability AI Assistant is a large language model (LLM) integration that helps Elastic Observability users by adding context, explaining errors and messages, and suggesting remediation in the {observability} interface. A connector for OpenAI or Azure OpenAI Service powers the Observability AI Assistant. 
```

Comment:
```suggestion
The Observability AI Assistant is a large language model (LLM) integration that helps Elastic Observability users by adding context, explaining errors and messages, and suggesting remediation in the {observability} interface. The Observability AI Assistant can be connected to either the OpenAI or Microsoft Azure LLM services. 
```

cc @dgieselaar Does this wording make more sense, as we're not using the connector framework?

Reply:
yes, definitely